x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/_legend.scss

```diff
@@ -28,11 +28,20 @@
   display: 'inline-block';
 }
 
-.mlJobMapLegend__inferenceModel {
+.mlJobMapLegend__trainedModel {
   height: $euiSizeM;
   width: $euiSizeM;
   background-color: '#FFFFFF';
```

Comment:
I lack the full context here, but if you'd like this to remain always white you can use the `$euiColorGhost` variable.

Reply:
Thanks for the suggestion - updated in 0e95f8993b8184dfdf3f02eaad3b476eda9ac79a